src/labels/CustomLabels.labels

```diff
@@ -9875,6 +9875,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Error shown when errors must be removed before something else can occur.</shortDescription>
         <value>Clear errors before proceeding.</value>
     </labels>
+    <labels>
+        <fullName>stgConcurrentBatchError</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```

Comment:
@diego-caballero , all custom labels should be protected: true. Can you please make the change.

Reply:
Thank you for catching that @RadSFDeveloper!